backend/db/migrations/20200722203947-addColumnsToTransactions.js

```diff
@@ -0,0 +1,44 @@
+
+const tableName = 'transactions'
+
+const { TransactionTypes, TransactionStatuses } = require('../../enums')
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.dropTable(tableName) // Note: this table was empty so it's fine to drop it.
+      await queryInterface.createTable(tableName, {
+        id: { type: Sequelize.INTEGER, autoIncrement: true, primaryKey: true },
+        shop_id: Sequelize.INTEGER,
+        network_id: Sequelize.INTEGER,
+        wallet: Sequelize.STRING,
```

Comment:
What is "wallet"?

Reply:
bad naming on my part. it's the wallet address the transaction was sent from. let me rename to "from". and might as well add a "to" column in there.

Reply:
I might recommend not using `from` as it's a reserved SQL keyword, but something more clear would be great.

Reply:
Good point. Used column names fromAddress and toAddress